pkg/log/audit.go

```diff
@@ -0,0 +1,50 @@
+package log
+
+import (
+	"fmt"
+	"strings"
+
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+)
+
+const (
+	lineLength = 40
+
+	messageSuccess = "SUCCESS"
+	messageSkipped = "SKIPPED"
+	messageFailed  = "FAILED " // leave the trailing space for consistent lengths
+)
+
+// Audit displays a message to the user. This shouldn't be used for debug logging purposes; all
+// messages passed in here should be user-readable.
+func Audit(message string) {
+	fmt.Println(message)
```

Comment:
What do you think about using the log package from the standard library? It includes the timestamp in a user friendly manner:
```
2023/11/30 09:27:04 Generating image customization components...
2023/11/30 09:27:04 Users ........................ [SUCCESS]
2023/11/30 09:27:04 Rpm .......................... [SKIPPED]
2023/11/30 09:27:04 Custom Scripts ............... [SKIPPED]
2023/11/30 09:27:04 Building ISO image...
2023/11/30 09:27:08 Image build complete!
```

Reply:
Generally speaking, I don't have strong opinions of how this function does it's thing. The real motivation behind this PR is that we have all of the code doing the same thing. Then we can dork with the details of how this function is implemented all we want.

I'll take a look at the log package. I'm certainly not hell bent on using `Println` and figured that we'd eventually make this at very least configurable to write to different options.